packages/yoga/src/Drawer/web/Drawer.jsx

```diff
@@ -36,6 +36,11 @@ function Drawer(props) {
 
 Drawer.propTypes = {
   children: node.isRequired,
+  zIndex: number,
+};
+
+Drawer.defaultProps = {
+  zIndex: 3,
```

Comment:
You don't need this default here right? It is already in the `Dialog`. It will fallback to that.

Reply:
Yeah i need, because this property is not required and because of that, you need to have a defaultProps for it.
If i remove this defaultProps eslint will give me this error: 
`propType "zIndex" is not required, but has no corresponding defaultProps declaration.`

Reply:
Ah ok :) Understood. As a suggestion you can also do this to make it "explicit" for the code reader:

```suggestion
  zIndex: Dialog.propTypes.zIndex,
```